xbmc/settings/VideoSettings.h

```diff
@@ -61,8 +61,13 @@ class CVideoSettings
   bool m_StereoInvert;
   int m_VideoStream;
 
-  static int GetNextViewMode(int viewMode);
-  static int GetViewModeStringIndex(int viewMode);
+  typedef struct
+  {
+    const int stringIndex;
+    const int nextViewMode;
```

Comment:
And exactly this is not good. You make this class dependent on the representation. VIdeoSettings should not care about the method the UI sets the mode